src/content/en/fundamentals/instant-and-offline/_toc.yaml

```diff
@@ -3,4 +3,6 @@ toc:
     section:
     - title: The Offline Cookbook
       path: /web/fundamentals/instant-and-offline/offline-cookbook/
+    - title: Web Storage Overview
```

Comment:
I'd suggest just titling this 'Web Storage' for now, then when we add some other stuff, we'll re-title it 'Overview'